### PR TITLE
Added code for L1 loss and applied uniform code formatting

### PR DESCRIPTION
**Pull request by rajaswa**
1. Added code for l1 loss
2. Added resources for l1 loss
3. Applied black code formatting across the repository

### CODE REVIEW

File: losses/l1/L1.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    # L1 distance is the reduced form of the difference array
+    if reduction == "sum":
+        # Reduction can be done by summing up all the values in the difference array (this is known as "L1-Loss")
+        l1_distance = np.sum(absolute_difference)
+        return l1_distance
+    elif reduction == "mean":
+        # Reduction can also be done by taking the mean (this is known as "Mean Absolute Error")
+        mean_absolute_error = np.mean(absolute_difference)
+        return mean_absolute_error
+    else:
```
Comment (alishdipani) [inline losses/l1/L1.py]: The function will never go into else condition as `reduction` has a default value, maybe there should be another keyword for returning `absolute_difference`

File: losses/l1/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Standard Implementations in ML Frameworks
+- [PyTorch: L1Loss](https://pytorch.org/docs/stable/generated/torch.nn.L1Loss.html)
+- [Keras: MeanAbsoluteError](https://keras.io/api/losses/regression_losses/#meanabsoluteerror-class)
+- [TF 2.3: MAE](https://www.tensorflow.org/api_docs/python/tf/keras/losses/MAE)
+
```
Comment (alishdipani) [inline losses/l1/README.md:10]: Also, add the formula if possible

**Review by alishdipani — CHANGES_REQUESTED** [review]

File: losses/l1/L1.py (hunk 1 above)
Comment (rajaswa) [reply losses/l1/L1.py]: `absolute_difference` can now be obtained by passing `reduction="None"`

File: losses/l1/README.md (hunk 2 above)
Comment (rajaswa) [reply losses/l1/README.md:10]: The formula has been added as an image

**Comment by alishdipani** [comment]
Looks good!
